modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -623,8 +623,11 @@ private void SetDiagonal(Vector3 diagonal)
         /// </summary>
         /// <param name="target">The position to look at.</param>
         /// <param name="up">The relative up direction.</param>
+        /// <param name="useModelFront">If specified, then the model is oriented in reverse,
+        /// towards the model front axis (+Z, Vector3.MODEL_FRONT),
+        /// which is more useful for orienting 3D models.</param>
         /// <returns>The resulting basis matrix.</returns>
-        public static Basis LookingAt(Vector3 target, Vector3 up)
+        public static Basis LookingAt(Vector3 target, Vector3 up, bool useModelFront = false)
```

Comment:
This change breaks binary compatibility, you need to keep the old method and add an overload. Also, while you are at it, it seems this method has a default value for the `up` parameter in Core. So you could add an overload with this method signature:

```csharp
public static Basis LookingAt(Vector3 target, Nullable<Vector3> up = null, bool useModelFront = false)
{
	up = up.HasValue ? up.Value : new Vector3(0, 1, 0);
```

I would also recommend to add `[EditorBrowsable(EditorBrowsableState.Never)]` to the old method to hide it from IntelliSense.

In order to avoid duplicating the documentation, keep the documentation in the new method and use `<inheritdoc>` in the old one, take a look at [Compat.cs](https://github.com/godotengine/godot/blob/b7c2fd2e9a7f01644e15fef86083a3e0e0221e4d/modules/mono/glue/GodotSharp/GodotSharp/Compat.cs) for an example.